apps/frontend/src/components/DifficultySelectPage.tsx

```diff
@@ -54,6 +62,25 @@ function DifficultySelectPage() {
       <Button variant="contained" onClick={matchUser}>
         Match
       </Button>
+
+      <Dialog open={isMatching} onClose={() => setIsMatching(false)}>
+        <DialogTitle>Matching User</DialogTitle>
+        <DialogContent>
+          <CountdownCircleTimer
```

Comment:
i think you may have missed committing the `package.json` with this dependency?

Reply:
Updated!